.github/pull_request_template.md

```diff
@@ -10,7 +10,8 @@ Contributors guide: https://github.com/TheAlgorithms/C-Plus-Plus/CONTRIBUTION.md
 <!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
 
 - [ ] Added description of change
-- [ ] Added tests and example, test passes
+- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#New-File-Name-guidelines)
+- [ ] Added tests and example, test must pass
```

Comment:
Do you have some existing tests in this repo that we could site as examples?

Do we want to have a check box for:
- [ ] Ran [cpplint](https://github.com/cpplint/cpplint), test must pass

Reply:
cpplint is run after the PR is created so the dev would have to edit the PR description after the checks have passed. Also working on a example test case and helper test cpp methods to be used for later and different pr.

Reply:
cpplint can (and should) be run on their local machine.